index.html

```diff
@@ -550,6 +550,138 @@
 template: inverse
 
 ## Serwer danych **Django**
+---
+.left-column[
+  ## Django
+  ### Co robi
+]
+.right-column[
+
+** Co robi serwer? **
+
+* Udostępnia wygenerowany na podstawie modeli interfejs [admina](https://eaterslab.herokuapp.com/admin/).
+
+* Udostępnia automatycznie wygenerowany na podstawie kodu dokument [API](https://eaterslab.herokuapp.com/doc).
+
+]
+
+???
+
+W interfejsie admina mamy dostęp do różnych elementów naszej bazy danych, takich jak 
+użytkownicy, kamery, stołówki ...
+
+API jest generowane automatycznie jednym z zainstalowanych dodatków do Django REST framework.
+
+---
+
+.left-column[
+  ## Django
+  ### Co robi
+  ### API
+]
+.right-column[
+
+** [API](https://eaterslab.herokuapp.com/doc) &ndash; między innymi: **
+
+* Odbiór przez aplikacje mobilne przeróżnych danych &ndash; przede wszystkim dane o stołówkach.
+
+* Wysyłanie przez kamery danych o zajętości stołówki.
+
+]
+
+???
+
+Pobawić się na /doc odbieraniem danych o przykładowych stołówkach i wysłać losowy camera event.
+
+---
+
+.left-column[
+  ## Django
+  ### Co robi
+  ### API
+  ### Baza Danych
+]
+.right-column[
+
+** Baza danych: PostgreSQL **
+
+* Szybka i rozbudowana.
+
+* Obsługuje duży i asynchroniczny dostęp do danych.
+
+]
+
+???
+
+Różne kamery mogą naraz wysyłać eventy, stołówki mogą w tym czasie pobierać dane...
+
+---
+
+.left-column[
+  ## Django
+  ### Co robi
+  ### API
+  ### Baza Danych
+  ### Dodatki
+]
+.right-column[
+
+** Dodatki **
+
+* [Django REST framework](https://www.django-rest-framework.org/)
+
+Budowanie API &ndash; serializacja danych, widoki API, autentykacja.
+
+* [drf-yasg](https://github.com/axnsan12/drf-yasg) &ndash; generacja dokumentu API.
+
+]
+
+???
+
+* Najważniejszym frameworkiem dodanym w naszym serwerze do Django jest Django REST framework.
+
+* Korzystając z natywnych modeli bazodanowych Django pozwala w prosty sposób tworzyć serializację a następnie
+widoki oraz autentykację. 
+
+* drf-yash tworzy /doc i .yaml
+
+---
+
+.left-column[
+  ## Django
+  ### Co robi
+  ### API
+  ### Baza Danych
+  ### Dodatki
+  #### Heroku
+]
+.right-column[
+
+** Heroku **
+
+* Hostuje serwer na chmurze.
+
+[build](https://eaterslab.herokuapp.com/) &ndash; [dev](https://eaterslab-dev.herokuapp.com/)
+
+* Buduje serwer przy każdym commit'cie.
```

Comment:
I'm not sure if we want to share this "implemmetation details" with audience. you can write somethin obout workflows that prepare the environment to run fresh server instance on user push to repo (not commit)